xrdsst/controllers/client.py

```diff
@@ -19,6 +19,11 @@ class Meta:
     def add(self):
         self.add_client(self.load_config())
 
+    @ex(label='add-service-description', help="Add client service description", arguments=[])
+    def add_description(self):
+        urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
```

Comment:
This ``urllib3`` insecure request warning can be removed as PR base is up-to-date with PR21 and this is only set once in application -- (for release it seems that one call will be removed too).